packages/mint-components/src/components/tax-and-cash/sqm-docusign-form/sqm-docusign-form.feature

```diff
@@ -0,0 +1,93 @@
+@owner:andy @author:andy
+Feature: Docusign Form
+
+  Background: A user has submitted their information in Step 1 and Step 2
+    Given a user is on the Docusign Form
+
+  @minutia
+  Scenario: Docusign URL is set in iframe on page load
+    When the Docusign Form loads
+    And the Docusign URL has been fetched
+    Then the URL set is set on the Docusign iframe
+    And the iframe loads based on the URL
+
+  @minutia
+  Scenario: Web-page fails when Docusign URL cannot be fetched
+    When the Docusign Form loads
+    And the Docusign URL cannot be fetched
+    Then the URL is not set on the Docusign iframe
+    And the iframe fails to load
+
+  @minutia
+  Scenario Outline: Participant not based in the US working with US brand selects participantType
+    When the page loads
+    Then <participantType> radio buttons appear
+    When they select a <participantType>
+    Then an <autoSelectedTaxForm> appears in the Docusign iframe
+
+    Examples: 
+      | participantType       | autoSelectedTaxForm |
+      | individualParticipant | W8-BEN              |
+      | businessEntity        | W8-BEN-E            |
+
+  @minutia @ui
+  Scenario Outline: Text copies change based on tax form participant fills out
+    When they are required to fill out <typeTaxForm>
+    Then the <taxFormSubHeader> displays with <typeTaxForm>
+    And the <taxFormDescriptionCopy> / <changeTaxFormCopy> changes depending on the <typeTaxForm>
+
+    Examples: 
```

Comment:
Just flagging the copy in these specs are going to change. The W9 one in particular is just not correct. Its for participants in the US period, it doesn't matter where the brand is